app/utils/rentry/wrapper.py

```diff
@@ -1,85 +1,88 @@
-import http.cookiejar
 import re
 import sys
-import urllib.parse
-import urllib.request
-from http.cookies import SimpleCookie
+import requests
 from json import loads as json_loads
 from PySide6.QtWidgets import (
     QDialog,
     QVBoxLayout,
     QLineEdit,
-    QPushButton,
     QMessageBox,
+    QPushButton,
 )
 from loguru import logger
 
-_headers = {"Referer": "https://rentry.co"}
+# Constants
+BASE_URL = "https://rentry.co"
+API_NEW_ENDPOINT = f"{BASE_URL}/api/new"
 
+_HEADERS = {"Referer": BASE_URL}
 
 class UrllibClient:
```

Comment:
I suggest renaming this class to something more descriptive, since `urllib` is not actually being used any more.

Reply:
Fixed